backend/src/modules/bill/resolvers/create-bill.ts

```diff
@@ -18,6 +19,10 @@ type Args = {
 type Context = {
   user: IUser;
 };
+const recurrenceSpanValid = (recurrenceSpan: string) => {
```

Comment:
let's follow the naming convention when the function returns a boolean

```suggestion
const isRecurrenceSpanValid = (recurrenceSpan: string) => {
```